fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1509,6 +1572,65 @@ Future<RangeResult> ReadYourWritesTransaction::getRange(const KeySelector& begin
 	return getRange(begin, end, GetRangeLimits(limit), snapshot, reverse);
 }
 
+Future<RangeResult> ReadYourWritesTransaction::getRangeAndFlatMap(KeySelector begin,
+                                                                  KeySelector end,
+                                                                  Key mapper,
+                                                                  GetRangeLimits limits,
+                                                                  Snapshot snapshot,
+                                                                  Reverse reverse) {
+	if (getDatabase()->apiVersionAtLeast(630)) {
```

Comment:
is 630 true? Not 700?

Reply:
Yes, the 630 copied from `getRange`. It is 630 because it handles special keys differently before that. Since the new API is added in 700, `getDatabase()->apiVersionAtLeast(630)` should aways be true. We could simplify the code by removing the else, but I would prefer keep it consistent with `getRange` so we can deduplicate in the future.